.github/workflows/ci.yml

```diff
@@ -92,9 +116,10 @@ jobs:
         run: ./install_wxpdfdoc.sh
 
       - name: Build lmi
-        run: make $coefficiency --output-sync=recurse install
+        run: make $coefficiency --output-sync=recurse
```

Comment:
I guess `make install` doesn't work under Linux? This is not a huge problem, but we should still have `Install lmi` step for MinGW case, i.e. I'd still like to test "make install" in the case in which it is supposed to work.